app/views/tracks/iterations/index/_iterations.html.haml

```diff
@@ -10,15 +10,33 @@
               %code
                 / TODO: 🤮
                 = iteration.submission.files.first.content
-          .analysis
-            - iteration.automated_feedback.each do |feedback|
-              .comment
-                .text= raw feedback.feedback_html
-                .byline
-                  = rounded_bg_img("https://avatars3.githubusercontent.com/u/135246?s=460&u=733b8575c8f0dbb5a52840c3efb2480b166c897a&v=4", "Erik SchierBOOM's uploaded avatar")
-                  %span
-                    by
-                    = link_to "Erik SchierBOOM", "#"
-                  .c-reputation{ 'aria-label': "#{123 * 1} reputation" }
-                    = icon(:reputation, "Reputation")
-                    123
+          .automated-feedback
+            - case iteration.automated_feedback_status
+            - when :pending
+              .automated-feedback-pending
+                = graphical_icon :spinner
+                %h3 We’re analysing your code for suggestions
+                %p This usually takes 10-30 seconds.
+
+            - when :present
+              .automated-feedback-present
+                .info
+                  = icon 'info-circle', "Extra Information"
+                  %p
+                    This is feedback written by our community, which may be of use.
+                    = link_to "Learn more.", "#"
+
+                - feedback = iteration.automated_feedback
+                .comment
+                  .c-textual-content.--small= raw feedback[:html]
+                  .byline
+                    = rounded_bg_img(feedback[:author][:avatar_url], "Erik SchierBOOM's uploaded avatar")
+                    = link_to "by #{feedback[:author][:name]}", feedback[:author][:profile_url]
+                    - if feedback[:author][:reputation]
+                      = render ViewComponents::Reputation.new(feedback[:author][:reputation])
+            - else
+              .automated-feedback-absent
+                = graphical_icon "graphic-conversation-1"
+                %h3 No auto suggestions? Try human mentoring.
+                %p Get real 1-to-1 human mentoring on the Leap exercise and start writing better C#.
+                = link_to "Get Mentoring", track_exercise_mentoring_index_path(iteration.track, iteration.exercise), class: 'btn-secondary'
```

Comment:
@kntsoriano This whole block should become a React Component basically. And we should listen to the `IterationChannel` for the changing `automated_feedback_status`.

You can get this working by doing the following:
1. Run up the tooling orchestrator and invoker, and make sure you have the Ruby Representer available in your dev env.
2. Create an iteration in the UI.
3. Go to http://local.exercism.io/maintaining/exercise_representations and enter some feedback by the iteration you've just made.
4. You should now see the feedback in the UI on the iterations page
5. Change states by running: `Submission.last.update(representation_status: :queued)` or `:generated` to toggle `automated_feedback_status` between `:pending` and `:present`, and toggle the presence of feedback (step 3) to switch `automated_feedback_status` between `:present` and `:none`.

---

If you need to rerun the representer, you can just do this in a `rails c`:
```reload!; Submission::Representation::Init.(Iteration.last.submission)```